### ISSUE TITLE
Startup screen encoding error (on windows?)

### ISSUE THREAD

**Issue by Tobotimus** [Cog-Creators/Red-DiscordBot#1048]
I'm getting an encode error when starting the bot.

# Output
```
(.env) C:\Users\tobyh\Documents\Repositories\Red-DiscordBot-v3\Bot>python -m redbot TARS.Dev --dev
Connected to Discord. Getting ready...
Loading packages...
Loaded packages: general, trivia
[31m
______         _           ______ _                       _  ______       _
| ___ \       | |          |  _  (_)                     | | | ___ \     | |
| |_/ /___  __| |  ______  | | | |_ ___  ___ ___  _ __ __| | | |_/ / ___ | |_
|    // _ \/ _` | |______| | | | | / __|/ __/ _ \| '__/ _` | | ___ \/ _ \| __|
| |\ \  __/ (_| |          | |/ /| \__ \ (_| (_) | | | (_| | | |_/ / (_) | |_
\_| \_\___|\__,_|          |___/ |_|___/\___\___/|_|  \__,_| \____/ \___/ \__|

[0m
Ignoring exception in on_ready
Traceback (most recent call last):
  File "C:\Users\tobyh\Documents\Repositories\Red-DiscordBot-v3\.env\lib\site-packages\discord\client.py", line 223, in _run_event
    yield from coro(*args, **kwargs)
  File "C:\Users\tobyh\Documents\Repositories\Red-DiscordBot-v3\Bot\redbot\core\events.py", line 117, in on_ready
    print(bordered(INFO, INFO2))
  File "C:\Users\tobyh\AppData\Local\Programs\Python\Python35\lib\encodings\cp850.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_map)[0]
UnicodeEncodeError: 'charmap' codec can't encode character '\u221a' in position 120: character maps to <undefined>
```

Screenshot: 
![image](https://user-images.githubusercontent.com/7438501/31926462-44f89d9e-b8da-11e7-9dbc-d69cf1ae3501.png)

# Environment

* Windows 10 version 1703
* Python 3.5.3
* Using Windows command prompt

**Comment by Redjumpman** [comment]
Yup this is a problem with command prompt. If you run it with PowerShell it will work. Command Prompt is being deprecated by Windows. All of the shortcuts that normal lead command prompt have been replaced by PowerShell. To workaround this, don't use command prompt if you can. Meanwhile, I'll be researching a way to fix this.

**Comment by Tobotimus** [comment]
Fixed in #1058
